fapi-client/src/main/scala/com/gu/facia/api/utils/CardStyle.scala

```diff
@@ -0,0 +1,101 @@
+package com.gu.facia.api.utils
+
+import com.gu.contentapi.client.model.Content
+import com.gu.facia.api.utils.ContentApiUtils._
+import com.gu.facia.client.models.MetaDataCommonFields
+
+object CardStyle {
+
+  def isExternalLink(content: Content, metaDataFields: MetaDataCommonFields): Boolean = (for {
+    snapType <- metaDataFields.snapType
+    href <- metaDataFields.href.orElse(content.safeFields.get("href"))
+  } yield snapType == "link" && ExternalLinks.external(href)) getOrElse false
+
+  def apply(content: Content, trailMetaData: MetaDataCommonFields): CardStyle = {
+    if (isExternalLink(content, trailMetaData)) {
+      ExternalLink
+    } else if (content.tags.exists(_.id == "news/series/hsbc-files")) {
```

Comment:
@adamnfish It has to exist somewhere unfortunately! And if we want to share this with apps, it has to be in this library!
Otherwise, we could let app not have `SpecialReports`, which is I doubt what people want!!
It's nasty.